app/src/main/java/com/example/whatsapp_clone/Model/Retrofit/HTTPClientDataSource.java

```diff
@@ -25,7 +25,6 @@ public class HTTPClientDataSource {
     private HTTPClientService service;
```

Comment:
we need to think of a different name for this, this is not a servicee

Reply:
it is a service , just not the android service